docs/UserGuide.md

```diff
@@ -26,26 +26,36 @@ Choose a topic from the table of contents to get started on your Wishful Shrinki
 
 1. Ensure your computer has Java `11` or above installed.
 
-2. Download the latest `wishfulShrinking.jar` from [here](https://github.com/AY2021S1-CS2103T-W10-2/tp).
+2. Download the latest `wishfulShrinking.jar` from [here](https://github.com/AY2021S1-CS2103T-W10-2/tp/releases).
 
 3. Copy the file to an empty folder you want to use as the _home folder_.
 
 4. **Double-click** the jar file to start the app OR start the app using **CLI** and type `java -jar <jar file name>.jar`.<br>
```

Comment:
should we specify our jar file name as wishfulShrinking.jar?

Reply:
what do you mean by this?